modules/openapi-generator/src/main/resources/swift5/libraries/alamofire/AlamofireImplementations.mustache

```diff
@@ -160,100 +160,20 @@ private var managerStore = SynchronizedDictionary<String, Alamofire.Session>()
         let validatedRequest = request.validate()
 
         switch T.self {
-        case is String.Type:
```

Comment:
When response type is void, getNonDecodableBuilder is used
- https://github.com/OpenAPITools/openapi-generator/blob/126e4061937fabd27a1ac020ca4dcfadb7b1e357/modules/openapi-generator/src/main/resources/swift5/api.mustache#L454
- https://github.com/OpenAPITools/openapi-generator/blob/126e4061937fabd27a1ac020ca4dcfadb7b1e357/modules/openapi-generator/src/main/resources/swift5/api.mustache#L163